atom/common/api/resources/windows_bindings.js

```diff
@@ -46,6 +47,57 @@ var binding = {
     ipc.send('chrome-windows-update', responseId, windowId, updateInfo)
   },
 
+  get: function (windowId, getInfo, cb) {
```

Comment:
please use `binding.registerCustomHook(function(bindingsAPI, extensionId)` for all of these. See tab_bindings.js for examples